packages/appium-tizen-tv-driver/lib/driver.js

```diff
@@ -156,21 +239,69 @@ class TizenTVDriver extends BaseDriver {
     });
     this.proxyReqRes = this.chromedriver.proxyReq.bind(this.chromedriver);
     this.proxyCommand = this.chromedriver.jwproxy.proxyCommand.bind(this.chromedriver);
-    this.jwpProxyActive = true;
+    this.#jwpProxyActive = true;
   }
 
-  async executeScript(script) {
+  /**
+   * Given a script of {@linkcode ScriptId} or some arbitrary JS, figure out
+   * which it is and run it.
+   *
+   * @template [TArg=any]
+   * @template [TReturn=any]
+   * @template {import('type-fest').LiteralUnion<ScriptId, string>} [S=string]
+   * @param {S} script
+   * @param {S extends ScriptId ? [Record<string,any>] : TArg[]} args
+   * @returns {Promise<S extends ScriptId ? import('type-fest').AsyncReturnType<ExecuteMethod<S>> : {value: TReturn}>}
+   */
+  async execute(script, args) {
+    if (TizenTVDriver.isExecuteScript(script)) {
+      log.debug(`Calling script "${script}" with arg ${JSON.stringify(args[0])}`);
+      return await this.executeMethod(script, [args[0]]);
+    }
+    return await /** @type {Promise<S extends ScriptId ? import('type-fest').AsyncReturnType<ExecuteMethod<S>> : {value: TReturn}>} */ (
+      this.executeChromedriverScript(script, args)
+    );
+  }
+
+  /**
+   * Execute some arbitrary JS via Chromedriver.
+   * @template [TReturn=any]
+   * @template [TArg=any]
+   * @param {((...args: any[]) => TReturn)|string} script
+   * @param {TArg[]} [args]
+   * @returns {Promise<{value: TReturn}>}
+   */
+  async executeChromedriverScript(script, args = []) {
+    const wrappedScript =
+      typeof script === 'string' ? script : `return (${script}).apply(null, arguments)`;
     return await this.chromedriver.sendCommand('/execute/sync', 'POST', {
-      script,
-      args: [],
+      script: wrappedScript,
+      args,
+    });
+  }
+
+  /**
+   * Execute some arbitrary JS via Chromedriver.
+   * @template [TReturn=any]
+   * @template [TArg=any]
+   * @param {((...args: any[]) => TReturn)|string} script
+   * @param {TArg[]} [args]
+   * @returns {Promise<{value: TReturn}>}
+   */
+  async executeChromedriverAsyncScript(script, args = []) {
```

Comment:
this seems like a straight up duplication of `executeChromeDriverScript` with just one char difference? `a`sync vs sync? maybe just pass an additional arg `async=false` to `executeChromedriverScript` and then `executeChromedriverAsyncScript` can be a one-liner